Shared/Coordinator/MainCoordinator+Launch.swift

```diff
@@ -18,14 +18,21 @@ extension MainCoordinator {
         case .success(let object):
             self.deepLink = object
 
-            if User.current().isNil {
+            guard let user = User.current() else {
                 self.runOnboardingFlow()
-            } else if let user = User.current(), !user.isOnboarded {
+                return
+            }
+
+            if !user.isOnboarded {
```

Comment:
Always push people through onboarding if they're not fully onboarded, regardless of activation status. We can make a little smarter later by skipping the welcome screen.